src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+// const rows = document.querySelectorAll('tr');
+const rows = document.querySelector('table').children[1].children;
```

Comment:
you can use native forms selectors instead of children
https://learn.javascript.ru/dom-navigation#dom-navigation-tables